Classes/Hook/UserSettings.php

```diff
@@ -65,6 +71,13 @@ class UserSettings
     /*─────────────────────────────────────────────────────────────────────────────*\
             Methods
     \*─────────────────────────────────────────────────────────────────────────────*/
+    public function __construct()
+    {
+        // Note: Dependency Injection does not work in this context
+        $this->objectManager = GeneralUtility::makeInstance(ObjectManager::class);
+        $this->dispatcher = GeneralUtility::makeInstance(Dispatcher::class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\TYPO3\CMS\Core\Utility\GeneralUtility' in method '__construct'.](https://app.codacy.com/manual/robinvonberg/cf_google_authenticator/pullRequest?prid=5687502)